ramp4-pcar4/ramp4-pcar4
# Fix maptip template to truncate after 3 lines

### Related Item(s)
#1902

### Changes
- [FEATURE] Adds `@tailwindcss/line-clamp` plugin which allows tailwind to truncate after `n` lines
- [FIX] Fixed maptip template to truncate after 3 lines

### Notes
New maptip template

![image](https://github.com/ramp4-pcar4/ramp4-pcar4/assets/34178949/ea7140f8-7b8f-42ad-a743-0b7cc5e659c5)



### Testing
Steps:
1. Load [Sample 42](https://ramp4-pcar4.github.io/ramp4-pcar4/hovertip-truncate-fix/demos/index-samples.html?sample=42)
2. Hover over the features and observe the maptip - there should be no truncation unless the text is longer than 3 lines


- - -
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/ramp4-pcar4/ramp4-pcar4/1934)
